<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QListWidgetItem:hover{
	background-color: rgb(255, 255, 255);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white</string>
   </property>
   <widget class="QLabel" name="lbl_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>599</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Статус синхронизации</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="input_area">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>271</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_draw">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>610</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_input_init_func">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>210</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>210</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>f(t) = </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_add_init_func">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>240</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QSlider" name="hs_param_L_val">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>310</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_param_L_from">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_param_L_to">
    <property name="geometry">
     <rect>
      <x>871</x>
      <y>310</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QSlider" name="hs_param_d_val">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>370</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_param_d_to">
    <property name="geometry">
     <rect>
      <x>871</x>
      <y>370</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_param_d_from">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>370</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSlider" name="hs_param_T_val">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>430</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_param_T_to">
    <property name="geometry">
     <rect>
      <x>871</x>
      <y>430</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_param_T_from">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>430</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_draw_matplotlib">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>610</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить(pyplot)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_set_end_time">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>470</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="placeholderText">
     <string>Время окончания</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cur_val_L">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>280</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущее значение параметра L: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cur_val_d">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>340</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущее значение параметра d: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cur_val_T">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>400</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущее значение параметра T: 0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_set_L">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>370</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>param L</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_set_d">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>410</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>param d</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_set_T">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>330</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>param T</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_param_h">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>440</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.001</string>
    </property>
    <property name="placeholderText">
     <string>Точность</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_del_init_func">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>240</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Время подсчета</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>530</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Шаг</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_h">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>530</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>0.001000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.001000000000000</double>
    </property>
    <property name="value">
     <double>0.001000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_t">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>530</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QListView" name="lw_input_area">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cur_val_k">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>460</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущее значение параметра k: 0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_param_k_to">
    <property name="geometry">
     <rect>
      <x>871</x>
      <y>490</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="hs_param_k_val">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>490</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_param_k_from">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>490</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_bar">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>570</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_n">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>530</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
